src/Microsoft.IdentityModel.JsonWebTokens/Json/JsonWebToken.PayloadClaimSet.cs

```diff
@@ -14,7 +14,12 @@ public partial class JsonWebToken
     {
         internal JsonClaimSet CreatePayloadClaimSet(byte[] bytes, int length)
```

Comment:
nit: Should this overload be removed so it's not accidentally used? Looks like it's used in tests only right now. (Same question about CreateHeaderClaimSet

Reply:
Considering its internal and not private I was hesitant to remove it.